### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jbuccola**

### CODE REVIEW

**Comment by jbuccola** [comment]
adding past-due homework

File: Johnny-jbuccola/homework_11/app.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          $('.images').prepend('<img class="image" src=' + element.image_url + ' />')
+          });
+        } 
+        else {
+          alert("There was an error of some sort.");
+        }
+      });
+    });
+  } else {
+    $('.images').append("Sorry your browser sucks");
```
Comment (Borgaard) [inline Johnny-jbuccola/homework_11/app.js:43]: Classy, haha.

File: Johnny-jbuccola/homework_11/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <header class="container-fluid text-center">
+    <h1>Local<span>Landscapes</span></h1>
+  </header>
+
+  <div class="container-fluid sign-in-view">
+    <div class="row text-center">
+      <h3>Discover how beautiful your surroundings can be</h3>
+      <button class="btn btn-default">
+        <a href="#" id="login">Login to 500px</a>
+      </button>
```
Comment (Borgaard) [inline Johnny-jbuccola/homework_11/index.html:24]: Consideration for later when we work with auth functionality: 
When a user is logged in, what happens to this `.login` id? Would you use templates to hide/show a login button depending on whether the user is authenticated? Would you use vanilla JS/jQuery to build the page upon determining authentication status?

**Comment by jbuccola** [comment]
Adding feedr project

**Pull request merged**